libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -1278,11 +1334,14 @@ export class ParentalLeaveService extends BaseTemplateApiService {
       // There has been case when island.is got Access Denied from AWS when sending out emails
       // This try/catch keeps application in correct state
       try {
+        const { residenceGrantFiles } = getApplicationAnswers(
+          application.answers,
+        )
+        if (residenceGrantFiles) return
```

Comment:
What happen to the application if applicant applied for residenceGrant and then apply for new period?
Does employer still get email for approval new period?

Reply:
No this needs fixing. I think I will clear files after it has been sent

Reply:
This is fixed